cpp/include/kvikio/shim/cufile.hpp

```diff
@@ -128,10 +122,16 @@ class cuFileAPI {
                 << std::endl;
     }
   }
+#else
+  cuFileAPI() { throw std::runtime_error("KvikIO not compiled with cuFile.h"); }
```

Comment:
```suggestion
  cuFileAPI() { throw std::runtime_error(CUFILE_ERRSTR(0)); }
```

To OAOO the error message.